travis/composer_check.py

```diff
@@ -0,0 +1,85 @@
+import argparse
+import os
+import json
+
+# the scripts current working directory. Used in defining default values for
+# this script
+cwd = os.getcwd()
```

Comment:
The python file open will default to the current directory if you just specify a filename so there is no need to explicitly pull in the cwd and prepend it to the filename.